src/mugs.js

```diff
@@ -982,6 +982,10 @@ define([
                 lstring: 'Comment',
                 visibility: 'visible',
                 widget: widgets.multilineText,
+                setter: function (mug, attr, value) {
+                    mug.p.set(attr, value);
+                    $('.fd-question-comment').text(value);
```

Comment:
ooh, that's brutal, coupling the mug (model) directly with the UI. What do you think of making the comment UI listen for the form `mug-property-change` event intsead?